app/views/admin/loan_questions/_loan_types.slim

```diff
@@ -0,0 +1,31 @@
+= f.input "loan_types" do
+  - if @loan_question.parent
+    / TODO: Use simple form yml to add contextual text for options
+    = f.input_field :override_associations, as: :select
+
+  / TODO: Table should be hidden or shown when modal first opens based on override_associations
+  table.table.loan-types-table
+    thead
+      tr
+        th colspan="2" Loan type
+        th Threshold Amount
+    tbody
+      = f.simple_fields_for :custom_field_requirements, @requirements do |sf|
```

Comment:
These feels non-standard as a way to pull in these items. If not non-standard, how to we incorporate this with simple form to pull in simple translations? Instead of using simple form, I duplicated the existing values in the database into yml to easily see what the loan types are and adjust their labels. The keys used in the database are numerical instead of something like "investment_loan".